example/lib/shared/player/knight.dart

```diff
@@ -131,15 +95,13 @@ class Knight extends SimplePlayer with Lighting, ObjectCollision, MouseGesture {
     );
   }
 
-  void actionAttackRange() {
-    if (stamina < 10) return;
-
+  void execRangeAttack(double angle, double attack) {
```

Comment:
`double attack` could be `double damage` to be consistent with the previous example